src/neo/SmartContract/ApplicationEngine.Binary.cs

```diff
@@ -23,6 +27,21 @@ protected internal StackItem BinaryDeserialize(byte[] data)
             return BinarySerializer.Deserialize(data, Limits.MaxStackSize, Limits.MaxItemSize, ReferenceCounter);
         }
 
+        protected internal string Itoa(BigInteger value)
```

Comment:
I think `base` parameter would be nice to have here (and for `Atoi`).

Reply:
Please take a look, octal and binary was not added, do you think that it's needed?

Reply:
To be fair I'd expect some regular 2-36 range to be supported here (like it's not a problem for [big.Int in Go](https://golang.org/pkg/math/big/#Int.Text)), but from a quick glance it looks like .NET's BigInteger doesn't make it easy even for 2-16 range (like in [this SO q/a](https://stackoverflow.com/questions/14048476/biginteger-to-hex-decimal-octal-binary-strings)). So maybe we can try to go with only 10 and 16 supported for now, it's not that hard to extend it in the future if need be.